<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRobots">
    <property name="title">
     <string>Robots</string>
    </property>
    <addaction name="actionAddRobot"/>
    <addaction name="actionDeleteRobot"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionResetCamera"/>
    <addaction name="separator"/>
    <addaction name="actionToggleGlobalAxis"/>
    <addaction name="actionToggleLocalAxes"/>
    <addaction name="actionToggleObjectCoordinates"/>
    <addaction name="actionToggleGrid"/>
   </widget>
   <addaction name="menuRobots"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddRobot"/>
   <addaction name="actionDeleteRobot"/>
   <addaction name="separator"/>
   <addaction name="actionResetCamera"/>
   <addaction name="actionToggleGlobalAxis"/>
   <addaction name="actionToggleGrid"/>
   <addaction name="separator"/>
   <addaction name="actionToggleLocalAxes"/>
   <addaction name="actionToggleObjectCoordinates"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>118</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QSlider" name="jointS2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="jointS3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSlider" name="jointS1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewSence">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/new.png</normaloff>:/resources/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New Scene</string>
   </property>
   <property name="toolTip">
    <string>Exits the program</string>
   </property>
  </action>
  <action name="actionLoadScene">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/open.png</normaloff>:/resources/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Scene</string>
   </property>
   <property name="toolTip">
    <string>Load Scene from a XML file...</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/save_as.png</normaloff>:/resources/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save Scene</string>
   </property>
   <property name="toolTip">
    <string>Save current Scene into XML file...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/exit.png</normaloff>:/resources/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddRobot">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/robot.png</normaloff>:/resources/icons/robot.png</iconset>
   </property>
   <property name="text">
    <string>Add Robot from a XML file...</string>
   </property>
  </action>
  <action name="actionOpenRobotCreate">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/new_robot.png</normaloff>:/resources/icons/new_robot.png</iconset>
   </property>
   <property name="text">
    <string>RobotCreate</string>
   </property>
   <property name="toolTip">
    <string>Open Robot Creator...</string>
   </property>
  </action>
  <action name="actionCollideAllLinks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/collide_all.png</normaloff>:/resources/icons/collide_all.png</iconset>
   </property>
   <property name="text">
    <string>All Links vs. all Obstacles</string>
   </property>
   <property name="toolTip">
    <string>Checks collisions between all Links and all Obstacles</string>
   </property>
  </action>
  <action name="actionCollideEndEffector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/collide_end.png</normaloff>:/resources/icons/collide_end.png</iconset>
   </property>
   <property name="text">
    <string>End Effector vs. all Obstacles</string>
   </property>
   <property name="toolTip">
    <string>Checks collisions just between end effector and all Obstacles</string>
   </property>
  </action>
  <action name="actionCollideNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/collide_none.png</normaloff>:/resources/icons/collide_none.png</iconset>
   </property>
   <property name="text">
    <string>No Collisions</string>
   </property>
   <property name="toolTip">
    <string>Disables collision cheking</string>
   </property>
  </action>
  <action name="actionAddRobotPath">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/robot_path.png</normaloff>:/resources/icons/robot_path.png</iconset>
   </property>
   <property name="text">
    <string>Add a Path to be followed by Robot...</string>
   </property>
   <property name="toolTip">
    <string>Adds a Path to be followed by the selected Robot</string>
   </property>
  </action>
  <action name="actionAddObstacle">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/objects.png</normaloff>:/resources/icons/objects.png</iconset>
   </property>
   <property name="text">
    <string>Add Obstacle from OBJ file...</string>
   </property>
  </action>
  <action name="actionAddPath">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/path.png</normaloff>:/resources/icons/path.png</iconset>
   </property>
   <property name="text">
    <string>Add Path...</string>
   </property>
   <property name="toolTip">
    <string>Adds a Path from a .txt file or from two points</string>
   </property>
  </action>
  <action name="actionResetCamera">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/reset_camera.png</normaloff>:/resources/icons/reset_camera.png</iconset>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="toolTip">
    <string>Resets the rotation and location of the camera to its default values.</string>
   </property>
  </action>
  <action name="actionToggleBlend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/alpha_blending.png</normaloff>:/resources/icons/alpha_blending.png</iconset>
   </property>
   <property name="text">
    <string>Alpha Blending</string>
   </property>
   <property name="toolTip">
    <string>Enables/Disables alpha blending for all objects in the scene</string>
   </property>
  </action>
  <action name="actionToggleGlobalAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/global.png</normaloff>:/resources/icons/global.png</iconset>
   </property>
   <property name="text">
    <string>Global Axis</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide Global reference frame</string>
   </property>
  </action>
  <action name="actionToggleLocalAxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/local.png</normaloff>:/resources/icons/local.png</iconset>
   </property>
   <property name="text">
    <string>Local Axes</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide local reference frame for all objects</string>
   </property>
  </action>
  <action name="actionToggleObjectCoordinates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/coordinates.png</normaloff>:/resources/icons/coordinates.png</iconset>
   </property>
   <property name="text">
    <string>Object Coordinates</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide the lines of spatial coordinates for all objects</string>
   </property>
  </action>
  <action name="actionToggleGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/grid.png</normaloff>:/resources/icons/grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide the base Grid</string>
   </property>
  </action>
  <action name="actionDeleteRobot">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/resources/icons/delete.png</normaloff>:/resources/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Robot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
